### PR TITLE
Add security audit doc

### PR DESCRIPTION
**Pull request by duglin**
Signed-off-by: Doug Davis <dug@microsoft.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -109,6 +109,10 @@ native ecosystem by making our systems interoperable with CloudEvents.
 If there is a security concern with one of the specifications in this
 repository please [open an issue](https://github.com/cloudevents/spec/issues).
 
+A security assessment was performed by
```
Comment (duglin) [inline README.md:112]: @JieDing any chance you could provide me the Chinese version of this text so I can update https://github.com/cloudevents/spec/blob/main/languages/zh-CN/README.md

Comment (JieDing) [reply README.md:112]: [Trail of Bits](https://www.trailofbits.com/) 于 2022 年 10 月对本项目进行了安全评估。详情参见[完整报告](docs/CE-SecurityAudit-2022-10.pdf)。
@duglin

Comment (JieDing) [reply README.md:112]: The link for the repot is a relative path, that might be a problem when you directly copy Chinese text into https://github.com/cloudevents/spec/blob/main/languages/zh-CN/README.md

Comment (duglin) [reply README.md:112]: would it be bad to make it `../../docs/CE-SecurityAudit-2022-10.pdf` for the Chinese version?

Comment (duglin) [reply README.md:112]: Thanks for the translation

Comment (JieDing) [reply README.md:112]: > would it be bad to make it `../../docs/CE-SecurityAudit-2022-10.pdf` for the Chinese version?

Or we can directly use `https://github.com/cloudevents/spec/tree/main/docs/CE-SecurityAudit-2022-10.pdf`. Both looks fine to me.

**Review by jskeet — APPROVED** [review]

**Comment by duglin** [comment]
Approved on the Nov 10th call

**Pull request merged**